lib/responses.js

```diff
@@ -373,6 +373,29 @@ function expectText(fn, res) {
   }).catch(handleError(res));
 }
 
+/**
+ * Checks the archived property on the request object to determine if the
+ * request is attempting to archive the page. If true, checks the published
+ * property on the meta object of the page, by calling checkProps.
+ * @param {object} req
+ * @param {object} res
+ * @param {function} next
+ * @returns {void}
```

Comment:
I think this method returns either a Promise or void
```suggestion
 * @returns {Promise | void}
```

Reply:
Yep.